content/pages/nuttx/getting-started/index.md

```diff
@@ -0,0 +1,279 @@
+tags: ["NuttX"]
+authors:
+    - "tiago-medicci"
+    - "eren-terzioglu"
+    - "filipe-cavalcanti"
+---
+
+## Getting Started with NuttX and ESP32 Family
+
+### Apache NuttX RTOS
+
+According to the Apache NuttX website:
+<cite>[NuttX is a real-time operating system (RTOS) with an emphasis on standards compliance and small footprint. Scalable from 8-bit to 64-bit microcontroller environments, the primary governing standards in NuttX are Posix and ANSI standards.][1] </cite>
+
+NuttX is the 2nd most popular community-based RTOS (along with Zephyr in the 1st position):
+
+| Operating system | First commit | Governance    | License                    | Contributors | Pulse (jun10) |
+|------------------|--------------|---------------|----------------------------|--------------|---------------|
+| Zephyr           | 2014         | community     | Apache 2.0                 | 100+         | 942           |
+| NuttX            | 2007         | community     | Apache 2.0                 | 100+         | 135           |
+| RT-Thread        | 2009         | community     | Apache 2.0                 | 100+         | 67            |
+| RIOT             | 2010         | community     | LGPL2.1                    | 100+         | 71            |
+| Tyzen RT         | 2015         | Samsung       | Apache 2.0                 | 100+         | 36            |
+| myNewt           | 2015         | Community     | Apache 2.0                 | 100+         | 25            |
+| mbed OS          | 2013         | ARM           | Apache 2.0 or BSD-3 Clause | 100+         | 7             |
+| FreeRTOS         | 2004         | Richard Barry | MIT                        | 100+         | 6             |
+| Contiki-NG       | 2016         | community     | BSD-3 Clause               | 100+         | 4             |
+| CMSIS-5          | 2016         | ARM           | Apache 2.0                 | 100+         | 0             |
+| Azure-RTOS       | 2020         | Microsoft     | Microsoft Software License | 10+          | archived      |
+
+— <cite>[Table by Alin Jerpelea, presented on NuttX Workshop 2024][2]</cite>
+
+Its standards conformance (POSIX and ANSI, mostly) allows software developed under other OSes (under the same standards, such as software developed for Linux) to be easily ported to NuttX, enabling embedded applications to reuse tested applications. In such a sense, NuttX can be seen as the closest alternative to Linux for embedded software, providing interfaces similar to the ones used by Embedded Linux applications to SoCs that do not require running Linux directly.
+
+NuttX supports more than 300 boards from different architectures and its active community keeps providing support for newer devices and boards. Espressif devices are supported on NuttX!
+
+#### Official Documentation
+
+The primary source of information for NuttX is its documentation, which is accessible at https://nuttx.apache.org/docs/.
+
+### Supported Espressif Devices
```

Comment:
Please correct me if I am wrong, but _device_ is embedded usually means a _slave_ as opposed to host.

How about:

```suggestion
### Supported Espressif SoCs
```

Reply:
Fixed ;)